### PR TITLE
Add \@if from Rhost/MUX.

### PR DESCRIPTION
**Pull request by talvo**
Like `@switch`, but booleanier.

### CODE REVIEW

File: game/txt/hlp/penncmd.hlp [hunk 1]
```diff
@@ -2375,6 +2375,41 @@ See also: @hook, EVALUATION ORDER, warn_on_missing, unimplemented_command
...
+  @ifelse and @skip/ifelse are aliases for @if.
+
+  See 'help @if2' for examples.
+See also: @break, @switch, if(), BOOLEAN VALUES
+& @if2
+  Examples:
+    > @if 1=say Yes, say No
+	You say, "Yes"
+	
+	> @if 0=say Yes, say No
```
Comment (tkrajcar) [inline game/txt/hlp/penncmd.hlp]: Appears to be extra spaces here and to the rest of this change.

File: src/cmds.c [hunk 2]
```diff
@@ -185,6 +184,32 @@ COMMAND(cmd_break)
   }
 }
 
+COMMAND(cmd_ifelse)
```
Comment (tkrajcar) [inline src/cmds.c]: Should this be `cmd_if` since the canonical command as documented is `@if`?

**Comment by tkrajcar** [comment]
Not seeing the alias for `@if` here, notwithstanding my other comment :)

Other than that, looks good! :+1:

**Comment by talvo** [comment]
There is a very good reason for that; I forgot it.

**Pull request merged**